source/Ship.cpp

```diff
@@ -2023,7 +2023,7 @@ double Ship::OutfitScanFraction() const
 // Fire any primary or secondary weapons that are ready to fire. Determines
 // if any special weapons (e.g. anti-missile, tractor beam) are ready to fire.
 // The firing of special weapons is handled separately.
-void Ship::Fire(vector<Projectile> &projectiles, vector<Visual> &visuals)
+void Ship::Fire(vector<Projectile> &projectiles, vector<Visual> &visuals, bool ammoAlert)
```

Comment:
The name `ammoAlert` makes me think it will always play the alert if this is true, and won't if it is false. But, this only tells it if it should consider playing it, whether or not it should actually be played is checked separately.
I think `flagship` or `isFlagship` would be better, since that's what this actually represents.